tests/queries/0_stateless/02845_domain_rfc_support_ipv6.sql

```diff
@@ -0,0 +1,5 @@
+SELECT domainRFC('[2001:db8::1]:80') FORMAT CSV;
+SELECT domainRFC('[::200]:80') FORMAT CSV;
```

Comment:
Consider adding cases with errors, e.g. with open and not closed bracket, several brackets, empty string within brackets and similar that comes to your mind.

Reply:
ok